src/shadowbox/webpack.config.js

```diff
@@ -34,6 +34,10 @@ const config = {
     new webpack.DefinePlugin({'global.GENTLY': false})
   ],
   resolve: {extensions: ['.tsx', '.ts', '.js']},
+  // dtrace-provider is a transitive dep used for sampling profiling on certain (non-Linux)
```

Comment:
How do you know it's not being used?
Won't webpack ignore it if it's not being used?
Do you know what dependency is bringing it?